src/database/DataTypes/ItemBatch.js

```diff
@@ -21,6 +22,10 @@ import Realm from 'realm';
  * @property  {List.<TransactionBatch>}  transactionBatches
  */
 export class ItemBatch extends Realm.Object {
+  get isVaccine() {
+    return this.item?.isVaccine ?? false;
+  }
```

Comment:
same thing added to `StocktakeBatch`, `TransactionBatch`, `ItemBatch` `TransactionItem`, just all the things